shards/modules/core/math.hpp

```diff
@@ -602,6 +816,427 @@ MATH_UNARY_FLOAT_OPERATION(Floor, __builtin_floor, __builtin_floorf);
+  }
+
+  static SHOptionalString inputHelp() { return SHCCSTR("The input float or sequence of floats to calculate the cosine of."); }
+
+  static SHOptionalString outputHelp() { return SHCCSTR("Outputs the cosine of the input."); }
+};
+RUNTIME_SHARD_TYPE(Math, Cos);
+
+struct Tan : public UnaryFloatOperation<BasicUnaryOperation<TanOp, DispatchType::FloatTypes>> {
+  static SHOptionalString help() {
+    return SHCCSTR("This shard calculates the tangent of the given input (tan(x)), where the input x is the angle in radians.");
+  }
+
+  static SHOptionalString inputHelp() { return SHCCSTR("The input float or sequence of floats to calculate the tangent of."); }
+
+  static SHOptionalString outputHelp() { return SHCCSTR("Outputs the tangent of the input."); }
+};
+RUNTIME_SHARD_TYPE(Math, Tan);
+
+struct Asin : public UnaryFloatOperation<BasicUnaryOperation<AsinOp, DispatchType::FloatTypes>> {
+  static SHOptionalString help() {
+    return SHCCSTR("This shard calculates the inverse sine of the given input (asin(x)), where the input x is the sine value.");
+  }
+
+  static SHOptionalString inputHelp() {
+    return SHCCSTR("The input float or sequence of floats to calculate the inverse sine of.");
+  }
+
+  static SHOptionalString outputHelp() { return SHCCSTR("Outputs the angle in radians whose sine is the input value."); }
+};
+RUNTIME_SHARD_TYPE(Math, Asin);
+
+struct Acos : public UnaryFloatOperation<BasicUnaryOperation<AcosOp, DispatchType::FloatTypes>> {
+  static SHOptionalString help() {
+    return SHCCSTR(
+        "This shard calculates the inverse cosine of the given input (acos(x)), where the input x is the cosine value.");
+  }
+
+  static SHOptionalString inputHelp() {
+    return SHCCSTR("The input float or sequence of floats to calculate the inverse cosine of.");
+  }
+
+  static SHOptionalString outputHelp() { return SHCCSTR("Outputs the angle in radians whose cosine is the input value."); }
+};
+RUNTIME_SHARD_TYPE(Math, Acos);
+
+struct Atan : public UnaryFloatOperation<BasicUnaryOperation<AtanOp, DispatchType::FloatTypes>> {
+  static SHOptionalString help() {
+    return SHCCSTR(
+        "This shard calculates the inverse tangent of the given input (atan(x)), where the input x is the tangent value.");
+  }
+
+  static SHOptionalString inputHelp() {
+    return SHCCSTR("The input float or sequence of floats to calculate the inverse tangent of.");
+  }
+
+  static SHOptionalString outputHelp() { return SHCCSTR("Outputs the angle in radians whose tangent is the input value."); }
+};
+RUNTIME_SHARD_TYPE(Math, Atan);
+
+struct Sinh : public UnaryFloatOperation<BasicUnaryOperation<SinhOp, DispatchType::FloatTypes>> {
+  static SHOptionalString help() {
+    return SHCCSTR("This shard calculates the hyperbolic sine of the given input (sinh(x)), where the input x is the real number "
+                   "used in the exponential calculation (sinh(x) = (e^x - e^-x) / 2).");
```

Comment:
Too much Foreign math notation all over the place, let's use Shards notation or replace with words